ResearchKit/Consent/ORKConsentSection.m

```diff
@@ -169,6 +169,7 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
         ORK_DECODE_OBJ_CLASS(aDecoder, summary, NSString);
         ORK_DECODE_OBJ_CLASS(aDecoder, content, NSString);
         ORK_DECODE_OBJ_CLASS(aDecoder, htmlContent, NSString);
+        ORK_DECODE_OBJ_CLASS(aDecoder, contentRequest, NSURLRequest);
```

Comment:
Just so I am clear ( it is a Friday of a 3 day weekend) 
I am just going to run the tests. Not adding anything to them to check for my content.

Just main haute I understood you.

Or 

Should I author tests to check for my NSURL property use.

J

Sent from my iPhone 6plus

> On May 22, 2015, at 4:25 PM, jwe-apple <notifications@github.com> wrote:
> 
> In ResearchKit/Consent/ORKConsentSection.m:
> 
> > @@ -169,6 +169,7 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
> >          ORK_DECODE_OBJ_CLASS(aDecoder, summary, NSString);
> >          ORK_DECODE_OBJ_CLASS(aDecoder, content, NSString);
> >          ORK_DECODE_OBJ_CLASS(aDecoder, htmlContent, NSString);
> > +        ORK_DECODE_OBJ_CLASS(aDecoder, contentRequest, NSURLRequest);
> Shouldn't have to write tests. Just make sure the tests pass. They'll fail if you add a property and do nothing.
> 
> —
> Reply to this email directly or view it on GitHub.
>